### ISSUE TITLE
[bug report] NVDA can't read iframe's title in web browser

### ISSUE THREAD

**Issue by khsbory** [nvaccess/nvda#6544]
Hello, I'm Hyongsop Kim in Korea.

In html, if I add a page contents with using iframe tag such as NV Access page, screen reader should read iframe page's title as a frame title.
If there is no page title, screen reader should read the page's url.
However NVDA doesn't read the title and only says 'frame'.
In that case, there is no way to let NVDA read frame title.
Please check this issue.

Thank you.

**Comment by feerrenrut** [comment]
Thanks for raising this issue @khsbory, however this seems to be a duplicate of #662. As such I will close this issue.
